internal/install/execution/terminal_reporter.go

```diff
@@ -155,6 +157,24 @@ func (r TerminalStatusReporter) UpdateRequired(status *InstallStatus) error {
 	return nil
 }
 
+func (r TerminalStatusReporter) printLoggingLink(status *InstallStatus) {
+	linkToLogging := ""
+	loggingMsg := "View your logs at the link below:\n"
+	statusesToDisplay := r.getRecipesStatusesForInstallationSummary(status)
+
+	for _, s := range statusesToDisplay {
+		if s.Status == RecipeStatusTypes.INSTALLED && s.DisplayName == "Logs integration" {
```

Comment:
We have an exported variable `LoggingRecipeName` that can we used in this conditional, but would refer to `s.Name` 

e.g. `if s.Status == RecipeStatusTypes.INSTALLED && s.Name == types.LoggingRecipeName { // ... }`